generators/app/templates/scripts/example.coffee

```diff
@@ -83,24 +83,24 @@ module.exports = (robot) ->
   #
   #   res.send 'OK'
   #
-  # robot.error (err, msg) ->
+  # robot.error (err, res) ->
   #   robot.logger.error "DOES NOT COMPUTE"
   #
-  #   if msg?
-  #     msg.reply "DOES NOT COMPUTE"
+  #   if res?
+  #     res.reply "DOES NOT COMPUTE"
   #
-  # robot.respond /have a soda/i, (msg) ->
+  # robot.respond /have a soda/i, (res) ->
   #   # Get number of sodas had (coerced to a number).
   #   sodasHad = robot.brain.get('totalSodas') * 1 or 0
   #
   #   if sodasHad > 4
-  #     msg.reply "I'm too fizzy.."
+  #     res.reply "I'm too fizzy.."
   #
   #   else
-  #     msg.reply 'Sure!'
+  #     res.reply 'Sure!'
   #
   #     robot.brain.set 'totalSodas', sodasHad+1
   #
-  # robot.respond /sleep it off/i, (msg) ->
+  # robot.respond /sleep it off/i, (res) ->
   #   robot.brain.set 'totalSodas', 0
   #   robot.respond 'zzzzz'
```

Comment:
I think it's actually supposed to be res.reply 'zzzzz'. Can you double check the hubot docs? I mention it because these examples were copied directly from there.